packages/core/package.json

```diff
@@ -25,14 +25,14 @@
     "clsx": "^2.1.1",
     "dso-toolkit": "workspace:^",
     "escape-string-regexp": "^5.0.0",
-    "focus-trap": "^7.6.0",
-    "i18next": "^23.16.2",
+    "focus-trap": "^7.6.1",
+    "i18next": "^23.16.5",
     "popper-max-size-modifier": "^0.2.0",
     "tabbable": "^6.2.0",
-    "uuid": "^11.0.2"
+    "uuid": "^11.0.3"
   },
   "devDependencies": {
-    "@stencil/angular-output-target": "0.7.1",
+    "@stencil/angular-output-target": "0.10.0",
```

Comment:
Deze moet je even goed in de gaten houden, want er bestaat een patch voor de versie 0.7.1 en ook mij is het niet helemaal duiduekijk wat die precies fixte, Thomas heeft mij dat wel is verteld, maar ik herinner me vooral, de opmerking, laat die versie maar even zo staan.

Reply:
Werkt de angular storybook nog?

Reply:
Je zegt dat je hem hebt teruggedraaid maar ik zie hier nog `0.10.0` staan